src/Template/Pages/home.ctp

```diff
@@ -170,6 +170,8 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
                     <ul><li>Everything CakePHP</li></ul></li>
                     <li><a href="http://plugins.cakephp.org">CakePHP plugins repo</a>
```

Comment:
As a side note: IMO this should be switched to the awesome list
This plugin site is not really maintained (to say the least) and does not really represent CakePHP in a very good way right now.